static/main.js

```diff
@@ -14,7 +14,7 @@ function init() {
     input.addEventListener("input", reprocess, false);
     permalink.addEventListener("click", setSearch);
 
-    const options = {displayMode: true, throwOnError: true, trust: true};
+    let options = {displayMode: true, throwOnError: true, trust: true};
```

Comment:
This can change back to `const` if you implement the comment below about modifying `options` in-place.